client_it_test.go

```diff
@@ -36,14 +36,55 @@ import (
 	"github.com/hazelcast/hazelcast-go-client/hzerrors"
 	"github.com/hazelcast/hazelcast-go-client/internal"
 	"github.com/hazelcast/hazelcast-go-client/internal/it"
-	"github.com/hazelcast/hazelcast-go-client/internal/proxy"
+	"github.com/hazelcast/hazelcast-go-client/internal/it/skip"
 	"github.com/hazelcast/hazelcast-go-client/logger"
 	"github.com/hazelcast/hazelcast-go-client/types"
 )
 
-var idGen = proxy.ReferenceIDGenerator{}
+func TestClient(t *testing.T) {
```

Comment:
We are able to run them all at once or one by one easily. Also, that makes it will make it easier to apply the same thing (e.g., parallelize them) easier.